parent/pom.xml

```diff
@@ -1216,8 +1216,16 @@
               <usePhrasedTestCaseClassName>true</usePhrasedTestCaseClassName>
               <usePhrasedTestCaseMethodName>true</usePhrasedTestCaseMethodName>
             </statelessTestsetReporter>
+            <consoleOutputReporter implementation="io.camunda.zeebe.ZeebeConsoleOutputReporter"></consoleOutputReporter>
```

Comment:
just noticed by this failure https://github.com/camunda/zeebe/actions/runs/3142205966/jobs/5105531759 that currently the unit test job doesn't use reruns and flaky test detection. Might be worth reenabling right now for consistency.

Reply:
Strange, as we had that in Jenkins :thinking: I would've thought we would keep it as well for GHA

Reply:
I suspect this got lost with the recent amount of changes, in this case the merge of the split unit tests jobs into one.

Reply:
I think I never enabled it for unit tests, even in the original GHA workflows :see_no_evil:

Reply:
I enabled it for unit, client, and smoke tests.